aten/src/ATen/native/native_functions.yaml

```diff
@@ -2805,8 +2805,24 @@
 
 - func: pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
 
+- func: _pixel_shuffle_cpu(Tensor self, int[] output_sizes, int upscale_factor) -> Tensor
+  dispatch:
+    CPU: _pixel_shuffle_cpu
```

Comment:
This looks weird. Why can't we just modify the existing pixel shuffle to look like this:

```
- func: pixel_shuffle(Tensor self, int upscale_factor) -> Tensor
  dispatch:
   CompositeImplicitAutograd: pixel_shuffle
   CPU: pixel_shuffle_cpu 
```

If you are concerned about having to recompute `output_sizes` in both implementations, consider making this a structured kernel: https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md

Reply:
COOL! Thanks!
I wasn't aware that pytorch has this `CompositeImplicitAutograd` dispatch key......

Reply:
updated with `CompositeImplicitAutograd` dispatch key!